openassessment/workflow/workflow_batch_update_api.py

```diff
@@ -259,7 +262,7 @@ def get_blocked_peer_workflows(course_id=None, item_id=None, submission_uuid=Non
 
     filters = {
         'created_at__lte': timezone.now() - datetime.timedelta(days=7),
-        'completed_at__isnull': True,
+        'grading_completed_at__isnull': True,
```

Comment:
You can also add `completed_at__isnull: False` here. If a learner hasn't completed the number of peer reviews they need to complete, their workflow isn't going to do anything when we update it.